static/app/views/settings/organizationDeveloperSettings/sentryApplicationDetails.tsx

```diff
@@ -277,6 +282,90 @@ export default class SentryApplicationDetails extends AsyncView<Props, State> {
     }
   };
 
+  getAvatarPreview = (avatarStyle: 'color' | 'simple') => {
+    const {app} = this.state;
+    const styleMap = {
+      color: {
+        size: 50,
+        title: t('Default Logo'),
+        description: t('The default icon for integrations'),
+      },
+      simple: {
+        size: 20,
+        title: t('Default Icon'),
+        description: t('This is an optional icon used for Issue Linking'),
+      },
+    };
+    return (
```

Comment:
You can do:
```tsx
if (!app) {
  return <React.Fragment />;
}
return (
  <AvatarPreview>
  ...
```

Reply:
Made it more explicit with the `if` statement, but I'm returning `null` instead